src/geqrf_qdwh_full.cc

```diff
@@ -0,0 +1,490 @@
+// Copyright (c) 2017-2022, University of Tennessee. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include "slate/slate.hh"
+#include "auxiliary/Debug.hh"
+#include "slate/Matrix.hh"
+#include "internal/internal.hh"
+
+namespace slate {
+
+// specialization namespace differentiates, e.g.,
+// internal::geqrf_qdwh_full from internal::specialization::geqrf
+namespace internal {
+namespace specialization {
```

Comment:
`namespace impl` instead of these two. Cf. geqrf.cc

Reply:
Right, it was copied from geqrf before impl namespace :)